tldw_Server_API/tests/MCP/test_mcp_tools_execute_authz.py

```
@@ -144,3 +144,105 @@ async def _seed():
+    db_file = tmp_path / "mcp_run_help.sqlite"
+    os.environ["DATABASE_URL"] = f"sqlite:///{db_file}"
+    os.environ["AUTH_MODE"] = "single_user"
+    os.environ.pop("MCP_MODULES", None)
+    os.environ.pop("MCP_MODULES_CONFIG", None)
+
+    _run(reset_db_pool())
+    reset_settings()
+    try:
+        get_config.cache_clear()  # type: ignore[attr-defined]
+    except Exception:
+        pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)